src/components/WarningModal.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import { Modal } from 'antd';
+
+interface Props {
+  isMobile: boolean;
+}
+
+export const WarningModal: React.FC<Props> = (props: Props) => {
+  const { isMobile } = props;
+  const [isVisible, setVisible] = useState(true);
+  return (
+    <>
+      {isMobile ? (
+        <Modal
+          visible={isVisible}
+          title="Mobile Warning"
+          closable={true}
+          centered={true}
+          onCancel={() => setVisible(false)}
+          footer={null}
+        >
+          <p>
+            For the best experience we recommend viewing the excess death tracker on your desktop
+            device. We currently working to improve the mobile site.
+          </p>
+        </Modal>
+      ) : null}
+    </>
+  );
```

Comment:
there's no need for conditional operator for this component since `<Modal />` will be visible only if the attribute `visible` is true.

Furthermore, if you needed to have some kind of condition to return one element, you don't need the ternary operator but use `&&` instead. Example:
```suggestion
  return isMobile && (
        <Modal
          visible={isVisible}
          title="Mobile Warning"
          closable={true}
          centered={true}
          onCancel={() => setVisible(false)}
          footer={null}
        >
          <p>
            For the best experience we recommend viewing the excess death tracker on your desktop
            device. We currently working to improve the mobile site.
          </p>
        </Modal>
  );
```
But in this case, since antd `<Modal />` has the `visible` attribute, you don't need any conditional operator for this.

Reply:
Makes sense